data/nfv/vsperf_sle12.4.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+zypper -q -n in -y $(echo "
```

Comment:
This is quite hard to read. I would suggest not having comments and just create a more compact list of packages

Reply:
I know, but the reason I am doing it this way is to respect the original script from VSPERF, e.g.: https://github.com/opnfv/vswitchperf/blob/master/systems/opensuse/42.3/build_base_machine.sh